content/docs/rfcs/36/README.md

```diff
@@ -451,7 +451,7 @@ For example:
 }
 ```
 
-### `extern char* waku_connect_peer(char* address, int timeoutMs)`
+### `extern char* waku_connect(char* address, int timeoutMs)`
```

Comment:
I would have gone for `connect_peer` instead as we know from context that this is a Waku method.